src/cclib/parser/gaussianparser.py

```diff
@@ -821,10 +821,11 @@ def extract(self, inputfile, line):
         # Matches "Step number  123", "Pt XX Step number 123" and "PtXXX Step number 123"
         if " Step number" in line:
             step = int(line.split()[line.split().index('Step') + 2])
+            if not hasattr(self, "optstatus"):
+                self.optstatus = []
+            self.optstatus.append(data.ccData.OPT_UNKNOWN)
```

Comment:
Could you combine this and the next two lines into an if/else block?

Reply:
Done in 81bc37874361714c77c80b69ac519114b4783fbb.